test/services/VatPartialBuilderSpec.scala

```diff
@@ -283,7 +283,7 @@ class VatPartialBuilderSpec extends ViewSpecBase with OneAppPerSuite with Mockit
         )
       }
 
-      "the user is in debit and has a Direct Debit set up" in new PaymentsSetup {
+      "the user is in debit and has a Direct Debit set up." in new PaymentsSetup {
```

Comment:
no need for this full stop

Reply:
Removed the full stop and amended previous commit.